process/TI-Gives+Gets.md

```diff
@@ -0,0 +1,52 @@
+# "Gives and Gets" for OpenSSF Technical Initiatives (TI)
+The OpenSSF has a large community of contributors and efforts that span the broad spectrum of open source security interests.  The Technical Initiatives (TIs) of the foundation are where our members collaborate and help craft unique solutions to addressing improving the security of the open source ecosystem.
+In exchange for meeting certain requirements, the TIs are eligible to receive an assortment of benefits and have access to the capabilities of the Foundation's resources.  The specific requirements and benefits (aka "Gives and Gets") for each level of maturity are documented below.  Based on the specific type of work the TI is focused on (e.g a software project vs. a specification or documentation-based effort) the requirements and benefits may slightly differ as applicable.
+
+
+## Sandbox level Gives & Gets
+
+|     Gives/Requirements             |                Gets/Benefits          |
```

Comment:
Nit: The table layout is a bit confusing because in the second table it doesn't line up similarly to the other two tables, e.g. just two lists. It makes it looks like the the individual gives and gets are potentially related to each other.

Reply:
Patches and formatting assistance welcome

Reply:
Fixed